### PR TITLE
Format and shorten action.yml description for marketplace

### PR DESCRIPTION
**Pull request by jwpereira**
Attempting to get into the marketplace now (woooo 🥳 ). First thing they flag is our descriptions being too long (needs to be < 125 characters). So, I've just grabbed the first sentence since it's straight to the point (the magic is really all @mw-hrastega).

As for the lack of punctuation at the end: that seems to be the trend with other actions 🤔

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 
-name: Run MATLAB 
-Tests description: >-
-  Run all tests in a MATLAB project and generate test artifacts. MATLAB includes
-  any files in your project that have a Test label. If your pipeline does not
-  leverage a MATLAB project, then MATLAB includes all tests in the the root of
-  your repository including its subfolders.
+name: Run MATLAB Tests
+description: >-
+  Run all tests in a MATLAB project and generate test artifacts
```
Comment (mw-hrastega) [inline action.yml:5]: How about this?

Run MATLAB and Simulink tests and generate artifacts

**Review by mw-hrastega — CHANGES_REQUESTED** [review]
I think we should remove "all", because support test selection.

File: action.yml (hunk 1 above)
Comment (acampbel) [reply action.yml:5]: Im on board with houman's suggestion

**Review by acampbel — APPROVED** [review]

**Review by mcafaro — APPROVED** [review]
+1 Houman

**Review by acampbel — APPROVED** [review]

**Review by mcafaro — APPROVED** [review]

File: action.yml [hunk 2]
```diff
@@ -2,60 +2,51 @@
...
-      on macOS platforms).
+      Path to write test results report in PDF format (not supported
+      on macOS)
     required: false
     default: ""
   test-results-junit:
     description: >-
-      JUnit-style test results Path to write test results report in JUnit XML
-      format.
+      JUnit-style test results path to write test results report in JUnit XML
```
Comment (mw-hrastega) [inline action.yml:31]: I think this can be shortened to:

Path to write test results report in JUnit XML format

File: action.yml [hunk 3]
```diff
@@ -2,60 +2,51 @@
...
-      Path to export Simulink Test Manager results in MLDATX format (requires
-      Simulink Test license and is supported in MATLAB R2019a or later).
+      Path to export Simulink Test Manager results in MLDATX format
     required: false
     default: ""
   code-coverage-cobertura:
     description: >-
       Cobertura code coverage Path to write code coverage report in Cobertura
-      XML format.
+      XML format
```
Comment (mw-hrastega) [inline action.yml:43]: To be consistent with other descriptions: 

Path to write code coverage report in Cobertura XML format

File: action.yml [hunk 4]
```diff
@@ -2,60 +2,51 @@
...
-      format.
+      JUnit-style test results path to write test results report in JUnit XML
+      format
     required: false
     default: ""
   test-results-simulink-test:
     description: >-
-      Path to export Simulink Test Manager results in MLDATX format (requires
-      Simulink Test license and is supported in MATLAB R2019a or later).
+      Path to export Simulink Test Manager results in MLDATX format
```
Comment (mw-hrastega) [inline action.yml:37]: For model coverage, you mention the release requirement. How about in here? Or you might leave out the parenthetic details altogether if they are not critical.

File: action.yml (hunk 3 above)
Comment (jwpereira) [reply action.yml]: Done! 😄

File: action.yml [hunk 5]
```diff
@@ -2,60 +2,51 @@
...
 inputs:
   source-folder:
     description: >-
-      Source code folder Location of the folder containing source code, relative
-      to the project root folder. The specified folder and its subfolders are
-      added to the top of the MATLAB search path. To generate a code coverage
-      report, MATLAB uses only the source code in the specified folder and its
-      subfolders. You can specify multiple folders using a colon-separated or a
-      semicolon-separated list.
+      Source code folder location of the folder containing source code
```
Comment (mw-hrastega) [inline action.yml:9]: Location of the folder containing source code

File: action.yml (hunk 4 above)
Comment (jwpereira) [reply action.yml:36]: Good catch; I'm not sure actually if we consider that critical 🤔 
@acampbel @mcafaro thoughts? 🤔

File: action.yml (hunk 5 above)
Comment (mw-hrastega) [reply action.yml:9]: Or

Location of the folder containing source code, relative to the project root folder

File: action.yml [hunk 6]
```diff
@@ -2,60 +2,51 @@
...
     required: false
     default: ""
   select-by-folder:
     description: >-
       Locations of the folders used to select test suite elements, relative to
-      the project root folder. To create a test suite, MATLAB uses only the
-      tests in the specified folders and their subfolders. You can specify
-      multiple folders using a colon-separated or semicolon-separated list.
+      the project root folder
```
Comment (mw-hrastega) [inline action.yml:15]: Location of the folder containing test files

or

Location of the folder containing test files, relative to the project root folder

**Review by mw-hrastega — CHANGES_REQUESTED** [review]
Almost there 👍

File: action.yml (hunk 4 above)
Comment (acampbel) [reply action.yml:34]: yeah i think
it can be in the doc, not needed in the one liner description

Comment (jwpereira) [reply action.yml:34]: Sounds good. I've cut the parenthetic data from `model-coverage-cobertura` to match this one 😄

File: action.yml [hunk 7]
```diff
@@ -1,66 +1,49 @@
...
-    required: false 
+      Test tag used to select test suite elements
+    required: false
     default: ""
   test-results-pdf:
     description: >-
-      Path to write test results report in PDF format (currently not supported
-      on macOS platforms).
-    required: false 
+      Path to write test results report in PDF format (not supported on macOS)
```
Comment (mw-hrastega) [inline action.yml]: It might be good to remove the parenthetic info here as well. They can find it in the doc.

**Review by mw-hrastega — APPROVED** [review]

File: action.yml (hunk 7 above)
Comment (jwpereira) [reply action.yml]: Done! 😄

**Pull request merged**